resources/profiles/AnkerMake.ini

```diff
@@ -76,18 +80,18 @@ min_skirt_length = 4
 notes = 
 only_retract_when_crossing_perimeters = 0
 ooze_prevention = 0
-output_filename_format = {input_filename_base}_{digits(layer_height,1,2)}mm_{filament_type[0]}_{printer_model}.gcode
+output_filename_format = {input_filename_base}_{layer_height}mm_{initial_filament_type}_{printer_model}.gcode
 overhangs = 1
 perimeter_acceleration = 2500
 perimeter_extruder = 1
-perimeter_extrusion_width = 0
-perimeter_generator = arachne
+perimeter_extrusion_width = 0.4
+perimeter_generator = classic
```

Comment:
I would be curious why the change from arachne to classic here. In my experience, arachne is a complete win for print quality